controllers/datadogagent/feature/apm/feature.go

```diff
@@ -363,6 +397,16 @@ func (f *apmFeature) manageNodeAgent(agentContainerName apicommonv1.AgentContain
 	}
 	managers.Port().AddPortToContainer(agentContainerName, apmPort)
 
+	// APM SSI Language Detection
+	if f.shouldEnableLanguageDetection() {
+
+		managers.EnvVar().AddEnvVar(&corev1.EnvVar{
```

Comment:
Does this env var need to be added to all containers? Can we limit it to only the containers that require it be set?